dotcom-rendering/src/web/components/SecondTierItem.tsx

```diff
@@ -8,11 +8,11 @@ import {
 	from,
 } from '@guardian/source-foundations';
 
```

Comment:
With regards to a flat file structure, our hands are tied by the use of the `.importable` pattern. Now we've moved this to the root of components, is it worth considering renaming this file to something like `MostViewedFooterSecondTierItem` so it's clear how this component is used/intended to be used?

Reply:
Thank you for seeing this! I'll do this now